### PR TITLE
Fix S3BucketName references after field renamed

### PR DESCRIPTION
**Pull request by upstart-swiss**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fix issues introduced in #20665 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by idovandijk — COMMENTED** [review]
Hi @upstart-swiss, thanks for the contribution.
Why do you need to change the name of the field? Changing field names is impossible. The result would be a broken playbook for existing users and a working one for those who install the pack from scratch.

**Review by idovandijk — COMMENTED** [review]
Please explain in the release notes that it breaks backward compatibility in the playbook because the incident field name is changing to a different name.

**Comment by upstart-swiss** [comment]
This isn't changing the field name, this is causing the cliname to match what it actually is when you install the pack. Try installing it, then check in the console. Also updating the playbook to reference that same cli name it installs with.

**Comment by idovandijk** [comment]
> This isn't changing the field name, this is causing the cliname to match what it actually is when you install the pack. Try installing it, then check in the console. Also updating the playbook to reference that same cli name it installs with.

I stand corrected, this is fine the way it is. Thanks for the fix.

**Review by idovandijk — APPROVED** [review]

**Review by YuvHayun — APPROVED** [review]

File: Packs/ContentManagement/IncidentFields/incidentfield-S3BucketName.json [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
     "caseInsensitive": true,
-    "cliName": "incidentconfigurationsetups3bucketname",
+    "cliName": "cicds3bucketname",
     "closeForm": false,
     "content": true,
     "editForm": true,
     "group": 0,
     "hidden": false,
-    "id": "incident_configurationsetup_s3bucketname",
+    "id": "cicd_s3_bucket_name",
```
Comment (upstart-swiss) [inline Packs/ContentManagement/IncidentFields/incidentfield-S3BucketName.json:13]: After reviewing an export, maybe this should be `incident_cicds3bucketname`

Comment (YuvHayun) [reply Packs/ContentManagement/IncidentFields/incidentfield-S3BucketName.json:13]: That's possible, you can change it.

**Pull request merged**